### ISSUE TITLE
Idle timeout issue with DXA web application

### ISSUE THREAD

**Issue by rajesh3200** [sdl/dxa-web-application-dotnet#19]
Occasionally, I am getting the following error, when we browse to the DXA Sample site homepage,

```
         _Server Error in '/' Application.Index was outside the bounds of the array._
```

Apparently, the issue gets resolved when we do an IISReset and refresh the homepage again.  Looks like it is the DummyCacheProvider which is causing the issue here.
Attached the site.log file for more details.
[Site.log.txt](https://github.com/sdl/dxa-web-application-dotnet/files/246346/Site.log.txt)

**Comment by majiccode** [comment]
Thanks for the log file rajesh. Seems like the issue is inside the type resolver. I've seen people mention what looks like the same problem with concurrent requests with version 6.14 of the Microsoft.OData.Client library.. Could you check what version you are running against ? I'll have a quick look inside their code to see if I can spot any possible issues too so I can eliminate it from my search.
Thanks
Paul.

**Comment by rajesh3200** [comment]
Hi Paul,

I am using DXA 1.4 Example Site

**Comment by rpannekoek** [comment]
SRQ-5404 (internal issue ID for tracking purposes)

**Comment by rpannekoek** [comment]
Fixed in DXA 1.6 / CIL 8.3.
